src/utilities/Button.tsx

```diff
@@ -7,11 +8,11 @@ type Props = ButtonHTMLAttributes<HTMLButtonElement> & {
   textColor: "dark-blue" | "white";
 };
 
-export const Button = ({ text, bg, textColor, type, ...props }: Props) => {
+export const Button = ({ text, bg, textColor, type, className, ...props }: Props) => {
```

Comment:
So is ref, but we're not using some other word for it, we're just passing prop called ref. I haven't heard about it not being recommended, I'm seeing className used as a prop all the time, even react (legacy) docs are using a variable called className to override it in a component: https://legacy.reactjs.org/docs/faq-styling.html.
Some random examples from across the web: https://stackoverflow.com/questions/40911516/how-to-override-classname-on-extended-component, https://github.com/styled-components/styled-components/issues/3179

Reply:
I found an explanation on airbnb: https://github.com/airbnb/javascript/tree/master/react#naming
It's just a good practice. So it depends on the team, if they want to follow this rule. In my projects, we always used this conventions.
I have the same opinion with `ref`. We can create a variable name called `wrapperRef`, `containerRef` etc.

Reply:
Ok, that convinces me! We might be using their eslint config later, so it's better to stick to their recommendations, I'll change it.

Reply:
Although it messes up types a bit, as className is a part of ButtonHTMLAttributes<HTMLButtonElement>, so we need to add classes "by hand".

Reply:
Honestly i will want to talk with you about that because i feel i missing something. For me using the attributes which comes from e.g. ButtonHTMLAttributes is ok 🤔

Reply:
Actually, I didn't notice before that we use all attributes from ButtonHTMLAttributes as props in  Button component. I've never seen such approach. Could you explain to me why we want to have dozens of props here? This seems like an exaggeration to me.